Zara Antonia-Maria/L4/3.6/main.c

```diff
@@ -0,0 +1,20 @@
+///palindrome
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    int nr,copy,newNr=0,lastDigit;
+    scanf("%d",&nr);
+    copy=nr;
+    do
+    {
+        lastDigit=copy%10;
+        newNr=newNr*10+lastDigit;
+        copy=copy/10;
+    }
+    while(copy!=0);
+    if(nr==newNr)printf("The number is a palindrome.");
```

Comment:
```
if(condition) { 
    // ... code
}
```

Hint: use `format use AStyle` thingy from CodeBlocks if you are using CodeBlocks